### ISSUE TITLE
Determining when to stop

### ISSUE THREAD

**Issue by gabrieldernbach** [lightly-ai/lightly#452]
Do you have practical recommendations for determining when to stop training? When working with unlabelled data people typically look for some convergence measure on the training loss.  What are your experiences here?

**Comment by philippmwirth** [comment]
Hi @gabrieldernbach 

Typically training longer will yield better results. The most popular papers train for 800 - 1000 epochs on ImageNet. 

As you can see in the plot below (from our CIFAR10 benchmarks), the loss will continuously decrease even after 800 epochs so getting a good metric to find convergence will be tricky.

![grafik](https://user-images.githubusercontent.com/65946090/127608834-12c11a8a-e060-454a-91df-17ac28815340.png)
